docs/modules/ROOT/pages/main/contribute.adoc

```diff
@@ -0,0 +1,52 @@
+= Contribution Guide
+
+This guide aims to make it easier for you to start contributing to the modelix project.
+
+== Setup
+
+=== Tools
+
+The following tools are used to develop modelix. You may want to install these on your machine before getting started.
```

Comment:
Lets stick to one sentence per line. Makes parsing and diffing easier in the future.

ps. please check rest of PR for this as well